jsdrupal/drupal-admin-ui
# issue-374: Type errors on /node/add/page & /node/add/recipe.

## Issue

https://github.com/jsdrupal/drupal-admin-ui/issues/374

## Testing instructions

Browse to node/add/recipe page observe in the console, the first error which is mentioned on the issue no longer appears.

For the second error mentioned on the issue, it is coming from the Drupal end and has been resolved. 
 I will raise another PR for it.
